content/10.diagnostics.md

```diff
@@ -137,20 +137,37 @@ The assay had 95% positive predictive agreement and 100% negative predictive agr
 The estimated limit of detection was 10 copies per &mu;l reaction, versus 1 copy per &mu;l reaction for the CDC assay.
 These results have been confirmed by other DETECTR approaches.
 Using real-time recombinase polymerase amplification (RT-RPA) for amplification, another group detected 10 copies of synthetic SARS-CoV-2 RNA per &mu;l of input within 60 minutes of RNA sample preparation in a proof-of-principle evaluation [@doi:10.1101/2020.02.29.971127].
+Using a similar approach, another group reported detection at 1 copy per &mu;l [@doi:10.1371/journal.pbio.3000978].
 The DETECTR protocol was improved by combining RT-RPA and CRISPR-based detection in a one-pot reaction that incubates at a single temperature, and by using dual CRISPR RNAs (which increases sensitivity).
-This new assay, known as All-In-One Dual CRISPR-Cas12a (AIOD-CRISPR), detected 4.6 copies of SARS-CoV-2 RNA per &mu;l of input in 40 minutes [@doi:10.1101/2020.03.19.998724].
-Another single-tube, constant-temperature approach using Cas12b instead of Cas12a achieved a detection limit of 5 copies/&mu;l in 40-60 minutes [@doi:10.1101/2020.04.10.023358].
-It was also reported that electric field gradients can be used to control and accelerate CRISPR assays by co-focusing Cas12-gRNA, reporters, and target [@doi:10.1101/2020.05.21.109637].
+This new assay, known as All-In-One Dual CRISPR-Cas12a (AIOD-CRISPR), detected 4.6 copies of SARS-CoV-2 RNA per &mu;l of input in 40 minutes [@doi:10.1038/s41467-020-18575-6].
+Another single-tube, constant-temperature approach using Cas12b instead of Cas12a achieved a detection limit of 5 copies/&mu;l in 40-60 minutes [@doi:10.1038/s41421-020-0174-y].
+It was also reported that electric field gradients can be used to control and accelerate CRISPR assays by co-focusing Cas12-gRNA, reporters, and target [@doi:10.1073/pnas.2010254117].
 The authors generated an appropriate electric field gradient using a selective ionic focusing technique known as isotachophoresis (ITP) implemented on a microfluidic chip.
 They also used ITP for automated purification of target RNA from raw nasopharyngeal swab samples.
 Combining this ITP purification with loop-mediated isothermal amplification, their ITP-enhanced assay to achieved detection of SARS-CoV-2 RNA (from raw sample to result) in 30 minutes.
 
+All these methods required upstream nucleic acid amplification prior to CRISPR-based detection. 
+They are relying on type V (Cas12-based) and type IV (Cas13-based) CRISPR systems.
+In contrast, type III CRISPR systems have the unique property of initiating a signalling cascade, which could boost the sensitivity of direct RNA detection. 
```

Comment:
```suggestion
In contrast, type III CRISPR systems have the unique property of initiating a signaling cascade, which could boost the sensitivity of direct RNA detection. 
```

Reply:
It would be great to add just one sentence on what the type III CRISPR system is otherwise its confusing why/how a signaling cascade is initiated if one does not know what the differences are.